annotate/module.ily

```diff
@@ -112,20 +112,37 @@
              #{ \once \override #item #'input-annotation = #props #}))))
          #{
           #tweak-command
-          #(if (assq-ref props 'footnote-offset)
-          ;; If offset present, add automatic footnote
+          #(cond
```

Comment:
I responded to this in the pull request - no reason for that conditional really; they can be applied to the same annotation without problems (that's in the example `balloon-text.ly` now as well), so 2e0bf6f fixes that. Thanks